packages/launchpad/src/checklist-item/style.scss

```diff
@@ -158,6 +163,10 @@
 	.badge {
 		background: var(--studio-blue-10);
 	}
+
+	.checklist-item__task-content[data-task="set_up_payments"] .badge {
```

Comment:
The colour of the badge could be defined other way around; we could instruct it within the task to "be green",  rather than greenines be defined by task ID?

Fine as is for this PR, not a blocker. :-)